applications/fishing-map/src/features/app/analytics.hooks.ts

```diff
@@ -1,40 +1,49 @@
 import { useEffect } from 'react'
 import { useSelector } from 'react-redux'
+import * as ua from 'react-ga'
 import { selectUserData } from 'features/user/user.slice'
+import { GOOGLE_UNIVERSAL_ANALYTICS_INIT_OPTIONS, IS_PRODUCTION } from 'data/config'
 
 const GOOGLE_UNIVERSAL_ANALYTICS_ID = process.env.REACT_APP_GOOGLE_UNIVERSAL_ANALYTICS_ID
 
 export const useAnalytics = () => {
   const userData = useSelector(selectUserData)
 
   useEffect(() => {
-    if (userData && GOOGLE_UNIVERSAL_ANALYTICS_ID && window.gtag) {
-      window.gtag('config', GOOGLE_UNIVERSAL_ANALYTICS_ID, {
-        user_id: userData.id,
-        custom_map: {
-          dimension1: 'userId',
-          dimension3: 'userGroup',
-          dimension4: 'userOrgType',
-          dimension5: 'userOrganization',
-          dimension6: 'userCountry',
-          dimension7: 'userLanguage',
-        },
+    if (GOOGLE_UNIVERSAL_ANALYTICS_ID) {
+      ua.initialize(GOOGLE_UNIVERSAL_ANALYTICS_ID, {
+        ...GOOGLE_UNIVERSAL_ANALYTICS_INIT_OPTIONS,
+      })
+      ua.set({
+        dimension1: 'userId',
+        dimension3: 'userGroup',
+        dimension4: 'userOrgType',
+        dimension5: 'userOrganization',
+        dimension6: 'userCountry',
+        dimension7: 'userLanguage',
       })
-      window.gtag('set', 'user_properties', {
-        user_id: userData.id,
-        user_group: userData.groups,
-        user_org_type: userData.organizationType,
-        user_organization: userData.organization,
-        user_country: userData.country,
-        user_language: userData.language,
+      // Uncomment to prevent sending hits in non-production envs
+      // if (!IS_PRODUCTION) {
+      //   ua.set({ sendHitTask: null })
+      // }
+    }
+  }, [])
+
+  useEffect(() => {
+    if (userData && GOOGLE_UNIVERSAL_ANALYTICS_ID) {
+      ua.set({
+        user_properties: {
+          user_id: userData.id,
```

Comment:
I've just realised this is using snake_case here but camelCase above... is this correct?

Reply:
Just to continue setting them as they were when we used gtag... I'll change it to camelCase